DigitalLearningSolutions.Web.AutomatedUiTests/AccessibilityTests/RegistrationJourneyAccessibilityTests.cs

```diff
@@ -106,6 +107,7 @@ public void Registration_by_centre_journey_with_set_password_has_no_accessibilit
             Driver.SelectDropdownItemValue("Answer1", "Principal Relationship Manager");
             Driver.FillTextInput("Answer2", "A Person");
             Driver.SelectDropdownItemValue("JobGroup", "1");
+            Driver.SelectRadioOptionById("HasProfessionalRegistrationNumber_No");
             Driver.SubmitForm();

```

Comment:
May as well try filling in a PRN here since we're not providing one in the other test.

Reply:
Made one of the tests attempt to input the PRN